app/web/features/search/FilterDialog.tsx

```diff
@@ -212,15 +240,39 @@ export default function FilterDialog({
               />
 
               <Select
-                id="can_host_status_filter"
-                value={hostingStatusFilter}
-                onChange={(e) => {
-                  setHostingStatusFilter(e.target.value as number);
-                }}
+                id="can_host_status_filter_1"
+                variant="outlined"
                 label={t("search:form.host_filters.hosting_status_field_label")}
+                multiple={true}
+                menuItems={true}
+                value={hostingStatusFilter}
+                onChange={(e) =>
+                  setHostingStatusFilter(e.target.value as number[])
+                }
```

Comment:
I don't think this is right. When they select an item, the value will be a single number. You're saying it's an array of numbers, but it's not.

Since it's a multiselect, I assume what you want to do is compare the `hostingStatusFilter`, if the value is there already, remove it, if it's not, add it.

Something like:

```javascript
  const handleHostingStatusFilterChange = (
    e: React.ChangeEvent<{ value: number }>
  ) => {
    if (hostingStatusFilter.includes(e.target.value)) {
      setHostingStatusFilter(
        hostingStatusFilter.filter((value) => value !== e.target.value)
      );
    } else {
      setHostingStatusFilter([...hostingStatusFilter, e.target.value]);
    }
  };

```

Reply:
you are absolutely right, the thing is, I got this example from the oficial Material UI page, they have it like this and it works (something that confuse me)

Reply:
![image](https://github.com/user-attachments/assets/6e447648-d0ed-4e04-98a4-665fde39b730)
![image](https://github.com/user-attachments/assets/22d759c4-7721-4196-9cb5-5739feb77f59)